contents/handbook/growth/marketing/messaging_framework.md

```diff
@@ -0,0 +1,87 @@
+
+## Audience pain-points
+Users within this audience may seek PostHog out because they need to:
+
+- Need to quickly get core product metrics (e.g. MAUs) onto a dashboard they can check regularly.
+- Gather data about how a product is used, especially drop-offs and retention. 
+- Identify technical issues within a product that are causing problems for users. 
+- Comply with data security regulations or avoid the risk of data leaks.
+- Integrate with existing tools, such as data warehouses. 
+
+## Audience ambitions
+Users may also seek PostHog out because they want to:
+
+- Combine quantitative and qualitative data about product usage.
+- Adopt a more experimental approach to product development.
+- Use open-source software, potentially with an eye towards customization.
+- Move forward quickly with immediate deployment. 
+
+## Positioning statement
+Our positioning statement is written using [Geoffrey Moore’s template](https://gist.github.com/JoshSmith/2041454). Our positioning statement is:
+
+_For product managers who need to understand their users, PostHog is the only open-source, all-in-one analytics platform that offers everything from funnel analysis and cohort tracking, to feature flags and session recording._ 
+
+_Unlike traditional tools, PostHog offers self-hosted deployments so that data doesn't need to be shared with anyone -- not even PostHog._
+
+## Alternative
+There are some alternative software which PostHog's users may consider. We split these into strong (e.g. regularly cited, competitive on multiple features) and weak (e.g. irregularly cited, competitive on few features). 
+
+_Strong:_ Heap Analytics, MixPanel, Amplitude, Pendo
+
+_Weak:_ HotJar (session recording), Optimizely (experimentation), Smartlook (analytics), FullStory (session recording & analytics), KissMetrics (analytics)
+
+## Messaging pillars
+Messaging pillars are our three core benefits or points of differentiation; points that define our product and our positioning; ideas which we own fully and reinforce regularly. The niches we dominate. 
+
+**All-in-one** — PostHog combines multiple types of analytics tools
+**Self-hosted** — PostHog can be deployed onto a clients’ own infrastructure
+**Extensible** — PostHog’s value can be extended beyond core metrics
+
+It may be tempting to ask of the second point: why we don’t describe this as ‘hosting flexibility’ or ‘hosting options’ given that PostHog can also be deployed on a cloud? The answer is that many alternative soplutions can be deployed on a cloud. Only PostHog can be self-hosted. 
+
+## Supporting messages
+Supporting messages are the key features/benefits which back up each pillar and can be used to prove its validity to customers. Where possible, these should tie to hard data. 
+
+**1 - All-in-one**
```

Comment:
How does the use of bold for headings vs. using an h3 (`###`) look?

Should the sub sections in italics be bullet points?

Reply:
I don't have a preference on this. Switched to H3. Left as no bullets just for speed.